tests/qvm_backend_test.py

```diff
@@ -390,6 +390,25 @@ def test_shots_bits_edgecases(qvm: None, quilc: None) -> None:
             assert res.get_counts() == correct_counts
 
 
+@pytest.mark.skipif(
+    skip_qvm_tests, reason="Can only run Rigetti QVM if docker is installed"
+)
+def test_gateset(qvm: None, quilc: None) -> None:
+    qc = get_qc("9q-square", as_qvm=True)
+    forest_backend = ForestBackend(qc)
+    a = [Qubit("node", i) for i in range(6)]
+
+    c = Circuit(0, 6)
+    for q in a:
+        c.add_qubit(q)
+    c.ISWAP(0.1, a[5], a[4])
+
+    c.measure_all()
+
+    h = forest_backend.process_circuit(c, 10)
+    forest_backend.get_result(h)
```

Comment:
It would be better to assert something about the result to check that the gate is being translated correctly.

Reply:
I have expanded the testcase here to check the result, see [641de02](https://github.com/CQCL/pytket-pyquil/pull/37/commits/641de0237edef79ce371fb6065364990f5a73cb7).

Was that what you wanted to see?